src/abstractarray.jl

```diff
@@ -199,6 +199,17 @@ reshape(a::Array, ::Size{S}) where {S} = SizedArray{Tuple{S...}}(a)
     return Expr(:tuple, (:(v[$i]) for i = N:(-1):1)...)
 end
 
+@eval @generated function Base.rot180(A::SMatrix{M,N}) where {M,N}
```

Comment:
Why is this `@eval @generated` instead of just `@generated`?

Reply:
Because I copy-pasted it out of the loop, sorry...